gulpfile.js

```diff
@@ -0,0 +1,96 @@
+/*eslint-env node */
+var gulp = require("gulp");
+var rename = require("gulp-rename");
+var request = require("request");
+var source = require("vinyl-source-stream");
+var gunzip = require("gulp-gunzip");
+var untar = require("gulp-untar");
+var gulpFilter = require("gulp-filter");
+var merge2 = require("merge2");
+var run = require("gulp-run");
+var install = require("gulp-install");
+
+gulp.task("default",
+    ["jshint", "jsonlint", "htmlhint", "jscs", "coffeelint", "csslint", "requirejs", "requirejs-text", "spromise", "eslint"],
+    function () {
+      console.log("Installed plugins");
+      return;
+});
+
+gulp.task("jshint", function () {
+    return gulp.src("./node_modules/jshint/dist/jshint.js")
+        .pipe(gulp.dest("./plugins/default/jshint/libs"));
+});
+
+gulp.task("jsonlint", function () {
+    return gulp.src("./node_modules/jsonlint/lib/jsonlint.js")
+        .pipe(gulp.dest("./plugins/default/jsonlint/libs"));
+});
+
+gulp.task("htmlhint", function () {
+    return gulp.src("./node_modules/htmlhint/lib/htmlhint.js")
+        .pipe(gulp.dest("./plugins/default/htmlhint/libs"));
+});
+
+gulp.task("jscs", function () {
+    return gulp.src("./node_modules/jscs/jscs-browser.js")
+        .pipe(gulp.dest("./plugins/default/jscs/libs"));
+});
+
+gulp.task("coffeelint", function () {
+    var coffeeScript = request("https://github.com/jashkenas/coffeescript/tarball/533ad8afe920b2dbf64ffb00efda45648242cc24")
```

Comment:
This can possibly be cleaned up by specifying the dependency in `package.json` (similar to how eslint is defined)